AppKit/CPTabView.j

```diff
@@ -527,9 +529,7 @@ var CPTabViewDidSelectTabViewItemSelector           = 1 << 1,
 
 - (void)mouseDown:(CPEvent)anEvent
 {
-    var segmentIndex = [_tabs testSegment:[_tabs convertPoint:[anEvent locationInWindow] fromView:nil]];
-
-    if (segmentIndex != CPNotFound && [self selectTabViewItemAtIndex:segmentIndex])
+    if ([_tabs testSegment:[_tabs convertPoint:[anEvent locationInWindow] fromView:nil]] !== CPNotFound)
         [_tabs trackSegment:anEvent];
 }
```

Comment:
Just one last thing: I think the "right" way to intercept events is to use -hitTest:
You override this method and return the segmented control (_tabs) if the mouse is over a segment and call super otherwise.

Reply:
I see in the tabView init that we set hitTests to NO for the segmented control.
What do you think about removing this and letting the segmented control doing its stuff ?
I think this was done to filter on selectTabViewItemAtIndex but as we removed this...

Reply:
Just tested and it works...